extension/manifest.json

```diff
@@ -11,10 +11,11 @@
     "content_scripts": [{
         "matches": ["*://*/*", "file:///*"],
         "js": ["content.js"],
-        "run_at": "document_end"
+        "run_at": "document_end",
+        "all_frames": true
     }],
     "background": {
         "scripts": ["background.js"]
     },
     "permissions": ["nativeMessaging"]
-}
```

Comment:
FYI: the missing newline was relevant at some stage, maybe it has changed. Will be included in next version, to be published about today.